pkg/controller/defaults/placements.go

```diff
@@ -90,6 +90,90 @@ var (
 			},
 		},
 
+		"osd-prepare": rook.Placement{
+			Tolerations: []corev1.Toleration{
+				corev1.Toleration{
+					Key:      NodeTolerationKey,
+					Operator: corev1.TolerationOpEqual,
+					Value:    "true",
+					Effect:   corev1.TaintEffectNoSchedule,
+				},
+			},
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
```

Comment:
Can you factor this to a helper method to avoid duplication of the whole antiaffinity spec?

Reply:
Noted!. as discussed, will plan to refactor it in a separate PR.